docs/jinja_filters.py

```diff
@@ -0,0 +1,72 @@
+import re
+
+from module_utils.fdm_swagger_client import SpecProp, PropName, PropType
+
+
+def camel_to_snake(text):
+    test_with_underscores = re.sub('(.)([A-Z][a-z]+)', r'\1_\2', text)
+    return re.sub('([a-z0-9])([A-Z])', r'\1_\2', test_with_underscores).lower()
+
+
+def get_link_to_model_page_by_name(model_name, display_name="object"):
+    return "[{}]{}".format(display_name, _get_link_path(model_name))
+
+
+def _get_link_path(model_name):
+    return "(/models/{}.md)".format(camel_to_snake(model_name))
+
+
+def show_type_or_reference(data_param_spec, api_spec=None):
```

Comment:
Please, document public functions here. Also, as these functions are side-effect free and quite well-scoped, consider writing examples in a form of [doctests](https://docs.python.org/3/library/doctest.html) as a part of the docstrings.

Reply:
fixed